docs/components/plugins/SnackEmbed.tsx

```diff
@@ -12,17 +10,25 @@ type Props = {
   preview?: boolean;
   theme?: string;
   style?: React.CSSProperties;
+  children?: string;
 };
 
-export default class SnackEmbed extends React.Component<Props> {
-  static contextType = PageMetadataContext;
-
-  componentDidMount() {
+const SnackEmbed: React.FC<Props> = ({
+  snackId,
+  name,
+  description,
+  platform,
+  preview,
+  theme,
+  style,
+  children,
+}) => {
```

Comment:
```suggestion
const SnackEmbed = ({
  snackId,
  name,
  description,
  platform,
  preview,
  theme,
  style,
  children,
}: Props) => {
```

We decided to not use FC in our codebase.

Reply:
Hey 👋  @Simek , thanks for the suggestions, I didn't know that the docs are being redesigned! So I think that leave us with the following options.
- I can update my PR with the suggestion you made if you give me green light 🟢  and merge it.
- Or I can just close it.

What do you think we should do?

Reply:
We can ship this conversion with the suggested changes, it won't collide, since we do not touch this component yet in the redesign process. 🙂

Reply:
Btw. You can find some of the new components already in `ui` directory, however most of them is not used on the website yet (besides Home and Error Pages), and there are stil few non finished components:
* https://github.com/expo/expo/tree/main/docs/ui/components

Reply:
I did the changes! 👍